Makefile

```
@@ -18,9 +18,7 @@ uv-lock-check: ## Check that the uv.lock file is in a good shape
 install-deps-test: ## Install all required dev dependencies needed to test the service, according to uv.lock
 	uv sync --group dev
 
-update-deps: ## Check pyproject.toml for changes, update the lock file if needed, then sync.
-	uv lock
-	uv sync --group dev
+update-deps: uv-lock-regenerate install-deps-test ## Regenerate lock files (with cooldown) and sync dev deps
```

checkmake (0.3.2)

[warning] Target "update-deps" should be declared PHONY.

(phonydeclared)